vars/dockerExecute.groovy

```diff
@@ -55,11 +55,11 @@ import groovy.transform.Field
     /**
       * Same as `dockerRegistryUrl`, but for the sidecar. If left empty, `dockerRegistryUrl` is used instead.
       */
-    'dockerSidecarRegistryUrl',
+    'sidecarRegistryUrl',
     /**
       * Same as `dockerRegistryCredentials`, but for the sidecar. If left empty `dockerRegistryCredentials` is used instead.
       */
-    'dockerSidecarRegistryCredentials',
+    'sidecarRegistryCredentials',
```

Comment:
```suggestion
    'sidecarRegistryCredentialsId',
```

Also applies to `dockerRegistryCredentials`

Reply:
Can this still be changed or is it (`dockerRegistryCredentials`) already productive?

Reply:
Good point, I'll have a look. I'll also add aliases for the other renamed params.